omnibus/cookbooks/omnibus-supermarket/attributes/default.rb

```diff
@@ -72,6 +72,7 @@
 default['supermarket']['install_directory'] = '/opt/supermarket'
 default['supermarket']['app_directory'] = "#{node['supermarket']['install_directory']}/embedded/service/supermarket"
 default['supermarket']['log_directory'] = '/var/log/supermarket'
+default['supermarket']['log_directory']['mode'] = '0700'
```

Comment:
Sadly, we won't be able to set a ['supermarket']['log_directory']['mode'] subkey when the parent key ['supermarket']['log_directory'] has a value type of String.

Reply:
sorry for the lack-of-sleep-induced unforced error
what about `default['supermarket']['log_directory_mode']`

Reply:
Cooould do that.

Now that I've also woken up a bit more, I thought of another option:
1. No configurable mode, so no new attribute needed.
2. The hard-coded mode changes from 0700 to 0740.
3. User accounts that get access to read Supermarket logs should be joined to the supermarket group by something outside of the Supermarket omnibus installer's internal chef-client run.